CONTRIBUTING.md

```diff
@@ -102,10 +102,25 @@ If you are developing under Windows, you can do the following to set this up:
 
 ## Developing with Eclipse
 
+### Install Eclipse with Lombok
+
 - Download Eclipse from [Eclipse 2019-03](https://www.eclipse.org/downloads/packages/release/2019-03/r), choosing "Eclipse IDE for Enterprise Java Developers". Note that 64-bit Eclipse doesn't work with 32-bit JRE/JDK (doesn't start without any message). There is no installer. To install Eclipse, just unzip your download to a directory of your choice.
+- Browse to https://projectlombok.org/.
+- On the top menu, choose "Download". Download version 1.18.12. You may need the link "older versions".
+- Run the .jar you downloaded. Under Windows you can double-click it.
```

Comment:
Hello Gerrit,

If you want to use Maven's download, then you have to find the downloaded .jar. This is not easy in Eclipse, because Eclipse does not sort the libraries. Then downloading manually is easier.